inc/plugin.class.php

```diff
@@ -922,6 +922,31 @@ function isActivated($directory) {
    }
 
 
+   /**
+    * Is a plugin updatable ?
+    *
+    * @param string $directory  Plugin directory
+    *
+    * @return boolean
+    */
+   function isUpdatable($directory) {
+      // Make a lowercase comparison, as sometime this function is called based on
+      // extraction of plugin name from a classname, which does not use same naming rules than directories.
+      $activated_plugins = array_map('strtolower', self::$activated_plugins);
+      if (in_array(strtolower($directory), $activated_plugins)) {
+         // If plugin is marked as activated, no need to query DB on this case.
+         return true;
```

Comment:
If plugin is active, it should mean that its state cannot be `NOTUPDATED`.
```suggestion
         return false;
```